### PR TITLE
OSDOCS-5743: Documented the 4.11.37 z-stream release notes

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.11

Issue:
[OSDOCS-5743](https://issues.redhat.com/browse/OSDOCS-5743)

Link to docs preview:
[Preview](https://58905--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-11-release-notes.html#ocp-4-11-37)

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

**Comment by dfitzmau** [comment]
Hi @cbippley . Would you be OK to perform a pre-peer-review on this PR?

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -3467,4 +3467,23 @@ $ oc adm release info 4.11.36 --pullspecs
 [id="ocp-4-11-36-updating"]
 ==== Updating
 
+All OpenShift Container Platform 4.11 users are advised that the only defect fixed in this release is limited to install time; therefore, there is no need to update previously installed clusters to this version.
```
Comment (JoeAldinger) [inline release_notes/ocp-4-11-release-notes.adoc:3470]:
```suggestion
All {product-title} 4.11 users are advised that the only defect fixed in this release is limited to install time; therefore, there is no need to update previously installed clusters to this version.
```

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -3467,4 +3467,23 @@ $ oc adm release info 4.11.36 --pullspecs
...
+
+[source,terminal]
+----
+$ oc adm release info 4.11.37 --pullspecs
+----
+
+[id="ocp-4-11-37-updating"]
+==== Updating
+
 All OpenShift Container Platform 4.11 users are advised that the only defect fixed in this release is limited to install time; therefore, there is no need to update previously installed clusters to this version.
```
Comment (JoeAldinger) [inline release_notes/ocp-4-11-release-notes.adoc:3489]:
```suggestion
To update an existing {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
```

**Review by JoeAldinger — COMMENTED** [review]
Something looks off in the commits. I'm not sure if you need to squash or if you accidentally committed something via GH.

**Comment by dfitzmau** [comment]
PR is troublesome, so I'm closing it in favour of https://github.com/openshift/openshift-docs/pull/59053

**Pull request closed without merging**